mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -117,6 +117,33 @@ def __eq__(self, other: object) -> bool:
             self._current_value == other._current_value
         )
 
+    def __lt__(self, other: object) -> bool:
```

Comment:
:thumbsup:

FWIW, I did some playing around with making the `Tunables` class more type aware with Generics:
https://github.com/microsoft/MLOS/compare/main...bpkroth:MLOS:mypy-tunable-values-tweak

I think for now it's an unnecessary and overly complicated dead end, so I wasn't gonna pursue it further, but I may keep it around in case we want it in the future.